design/www/app/common/lastusestime.service.js

```diff
@@ -0,0 +1,45 @@
+ (function() {
+     'use strict';
+     angular.module('starter')
+         .factory('lastUsesTimeService', lastUsesTimeService);
+
+     function lastUsesTimeService(lastUsesTimeFactory, timeStorage, $interval, $localStorage) {
+         return {
+             updateTime: function(userId) {
```

Comment:
Sir i used userId here for when user login first time.
as trello card says if user login then fire this api. so i used when user login api gives response userId, i directly used this userId. if i call only service without pass user id , then localStorage has no data related to user id. i saved all user data to localStorage when user login but service tell userId is undefined. so i used this.